API.md

```diff
@@ -674,6 +674,9 @@ Oplog or IPFS-Log is the underlying data structure used by the OrbitDB Stores to
 replicate and add operations to any of their stores. 
 
 ## `oplog.values`
+An Array of operations sorted in ascending order by their time.
+If an operation A is before B, A is before B in the Array.
```

Comment:
I'd try to leave the term 'time' out of this

Reply:
What is it ordered by if not time? Causal relationship?

Reply:
yes and a logical clock but its not necessary to go into that here.